nyxx.interactions/lib/src/Events/InteractionEvent.dart

```diff
@@ -9,44 +9,44 @@ class InteractionEvent {
   InteractionEvent._new(Nyxx client, Map<String, dynamic> rawJson) {
     this._client = client;
     interaction = Interaction._new(client, rawJson);
+
+    if(interaction.type == 1) {
+      this._pong();
+    }
   }
 
   /// Should be sent when you receive a ping from interactions. Used to acknowledge a ping.
-  Future Pong() {
+  Future _pong() {
```

Comment:
Add type of Future, in this casae should be `Future<void>` I belive

Reply:
done